cmd/snap/cmd_advise.go

```diff
@@ -205,11 +210,50 @@ func adviseViaAptHook() error {
 	return nil
 }
 
+type Snap struct {
+	Snap string
+	Version string
+	Command string
+}
+
+func dumpDbHook() error {
+	commands, err := advisor.DumpCommands()
+	if err != nil {
+		return err
+	}
+
+	commands_processed := make([]string, 0, len(commands))
+	var b[] Snap;
```

Comment:
`go fmt` would've fixed this :-)
Can't your editor run it as a save hook?

Reply:
you should add it as a git commit hook. something like this: https://golang.org/misc/git/pre-commit

Reply:
that's a different approach, but it's still something each dev needs to do AFAIK

Reply:
nope, systemd ships a git pre-commit hook.